GsaGH/Parameters/5_Results/PointResultGoo.cs

```diff
@@ -9,27 +9,37 @@ namespace GsaGH.Parameters
 {
   public class PointResultGoo : GH_GeometricGoo<Point3d>, IGH_PreviewData
   {
-    public PointResultGoo(Point3d point, IQuantity result, Color color, float size)
+    public PointResultGoo(Point3d point, IQuantity result, Color color, float size, int id)
     : base(point)
     {
       Result = result;
-      _mSize = size;
-      _mColor = color;
+      _size = size;
+      _color = color;
+      NodeId = id;
     }
 
-    private readonly float _mSize;
-    private readonly Color _mColor;
+    public int NodeId;
```

Comment:
variables, props, fields should be before constructor ;)

Reply:
can we move this in the next refactor round?